extensions/amp-carousel/0.1/base-carousel.js

```diff
@@ -138,9 +163,12 @@ export class BaseCarousel extends AMP.BaseElement {
    */
   setControlsState() {
     this.prevButton_.classList.toggle('amp-disabled', !this.hasPrev());
-    this.prevButton_.setAttribute('aria-disabled', !this.hasPrev());
+    this.prevA11yBtn_.classList.toggle('amp-disabled', !this.hasPrev());
+    this.prevA11yBtn_.setAttribute('aria-disabled', !this.hasPrev());
```

Comment:
since CSS will `display: none` these with `amp-disbaled`, adding `aria-disabled` is irrelevant.